lib/scrivener/html.ex

```diff
@@ -233,11 +242,17 @@ defmodule Scrivener.HTML do
   defp li_classes_for_style(paginator, page_number, :semantic) do
     if(paginator.page_number == page_number, do: ["active", "item"], else: ["item"])
   end
+  defp li_classes_for_style(_paginator, :ellipsis, :materialize), do: []
+  defp li_classes_for_style(paginator, page_number, :materialize) do
+    if(paginator.page_number == page_number, do: ["active"], else: ["waves-effect"])
```

Comment:
You may want to make it configurable to be able to change the wave effect color if you're going to include it...